tests/integ/modin/test_modin_stored_procedures.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import pytest
+
+try:
+    import modin.pandas as pd  # noqa: F401
+
+    is_pandas_available = True
+except ImportError:
+    is_pandas_available = False
+
+from snowflake.snowpark import Session
+from snowflake.snowpark.functions import sproc
+from tests.integ.modin.sql_counter import sql_count_checker
+from tests.utils import TestFiles, Utils
+
+pytestmark = [
+    pytest.mark.udf,
+]
+
+tmp_stage_name = Utils.random_stage_name()
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup(session, resources_path, local_testing_mode):
+    test_files = TestFiles(resources_path)
+    if not local_testing_mode:
+        Utils.create_stage(session, tmp_stage_name, is_temporary=True)
+        session.add_packages("snowflake-snowpark-python")
+    Utils.upload_to_stage(
+        session, tmp_stage_name, test_files.test_sp_py_file, compress=False
+    )
+
+
+@sql_count_checker(no_check=True)
```

Comment:
why skip checks?

Reply:
I was assuming if tests are run in stored procs, we aren't concerned with sql count checks. Should I add them?